File: Amazon/ServerSide/Amazon.Solution/Amazon.Core/Entities/WishlistItem.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Amazon.Core.Entities
{
    public class WishlistItem
    {
        public int Id { get; set; }

        [Required]
        public string ProductName { get; set; }
        [Required]
        public string Category { get; set; }

        [Range(0, double.MaxValue)]
        public decimal Price { get; set; }
        public string PictureUrl { get; set; }

        [Range(1, int.MaxValue)]
        public int Quantity { get; set; }
    }
}

File: Amazon/ClientSide/Amazon.ClientSide/src/app/Models/wish-list-item.ts
export class WishListItem {
    id:number
    productName:string
    category:string
    brand:string
    price:number
    pictureUrl:string
    quantity:number


    constructor(id: number, ProductName: string, Category: string,Brand: string = "none",Price:number,PictureUrl:string,Quantity:number)
    {
        this.id = id;
        this.productName = ProductName;
        this.category = Category;
        this.brand = Brand;
        this.price = Price;
        this.pictureUrl = PictureUrl;
        this.quantity = Quantity;
    }
}
